setup_wordpress.sh

```diff
@@ -4,3 +4,83 @@ set -e
 
 # Your code goes below ...
 echo 'This script should install and setup Wordpress'
+
+#Install nginx, mysql and php packages
+sudo apt update
+sudo apt upgrade -y
+sudo apt install nginx -y
+sudo apt install mysql-server -y
+sudo apt install php7.4-cli php7.4-fpm php7.4-mysql php7.4-json php7.4-opcache php7.4-mbstring php7.4-xml php7.4-gd php7.4-curl -y
+
+#Firewall
+sudo ufw enable
+sudo ufw allow 'Nginx Full'
+
+#Create database
+sudo mysql -e "CREATE DATABASE wpDB DEFAULT CHARACTER SET utf8 COLLATE utf8_unicode_ci;"
+sudo mysql -e "CREATE USER 'berna'@'localhost' IDENTIFIED BY 'Berna123';"
+sudo mysql -e "GRANT ALL ON wpDB.* TO 'berna'@'localhost';"
+sudo mysql -e "FLUSH PRIVILEGES;"
+
+#Wordpress install and permissions
+cd /tmp
+sudo wget https://wordpress.org/latest.tar.gz
```

Comment:
I think it's not necessary but i added just in case. Wget is a command that lets us download files from the web using protocols like http, https, ftp.

Reply:
I would recommend give programs as little privileges as possible. Here you give complete `root` privileges, so potentially, if wget is somehow "hacked" you give it big chance

Reply:
Done, i deleted sudo from wget